### ISSUE TITLE
missing temp file causing errors for past samples

### ISSUE THREAD

**Issue by alethomas** [koesterlab/snakemake-workflow-sars-cov2#157]
```
InputFunctionException in line 92 of /local/data/snakemake-workflow-sars-cov2/workflow/rules/generate_output.smk:
Error:
  WorkflowError:
    File results/2021-03-13/rki-filter/2021-03-13.txt cannot be opened, since it does not exist.
Wildcards:
  date=2021-03-13
Traceback:
  File "/local/data/snakemake-workflow-sars-cov2/workflow/rules/generate_output.smk", line 99, in <lambda>
  File "/local/data/snakemake-workflow-sars-cov2/workflow/rules/common.smk", line 29, in get_samples_for_date
  File "/home/alex/anaconda3/envs/snakemake/lib/python3.9/contextlib.py", line 117, in __enter__
```

**Comment by thomasbtf** [comment]
I think this is/will be addresed in f10f0afe9437d3329a40a320969f8af1f60ef5d3.

**Comment by thomasbtf** [comment]
And probably thus fixed with #158. Thus, I'll close this. Hopefully not just for the moment :)
